package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prezly/theme-kit-nextjs",
-  "version": "1.6.2",
+  "version": "1.6.3-0",
```

Comment:
Should this be committed? 🤔

Reply:
This was auto-pushed when publishing a release preview for testing. I think it's fine to commit it, since there is a package with this version on NPM